client/src/plugins/camunda-plugin/deployment-tool/DeploymentTool.js

```diff
@@ -138,6 +139,11 @@ export default class DeploymentTool extends PureComponent {
 
       await this.handleDeploymentSuccess(tab, deployment);
     } catch (error) {
+
+      if (!(error instanceof DeploymentError)) {
```

Comment:
What about `StartInstanceError`? Are they handled appropriately? I'm asking as we should have the same problem there.

Reply:
We have the same situation in [StartInstanceTool](https://github.com/camunda/camunda-modeler/blob/develop/client/src/plugins/camunda-plugin/start-instance-tool/StartInstanceTool.js#L229).

So:
* In this particular line of code, I would argue we should stick to only check for DeploymentError (note the implementation in [CamundaAPI](https://github.com/camunda/camunda-modeler/blob/develop/client/src/plugins/camunda-plugin/shared/CamundaAPI.js#L29) which would in particular throw DeploymentError
* For StartInstanceTool I would create another PR following the same schema (this PR aims at dealing with https://github.com/camunda/camunda-modeler/issues/2078)

WDYT?

Reply:
Makes complete sense. Thanks for clarifying.

Reply:
Keep in mind that the start instance tool uses the deployment tool under the hood. We still want to capture all business errors there.